ci-operator/step-registry/cloud-bulldozer/e2e-benchmarking/rds-core/cloud-bulldozer-e2e-benchmarking-rds-core-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+pushd /tmp
+
```

Comment:
You are missing some pre-reqs: https://github.com/kube-burner/kube-burner-ocp/tree/main?tab=readme-ov-file#core-rds-workloads

We need one host labeled as **_worker-dpdk_** and another one as **_worker-metallb_**. In openshift-qe/installer/bm/day2/label there is a step that you can reuse to label the nodes.